src/layouts/index.tsx

```diff
@@ -119,41 +106,69 @@ const Layout = ({ children }) => (
         <Footer>
         <Location>
           {({ location }) => {
-            const index = LINKSPATH.findIndex(path => path.path === location.pathname)
-            if(location.pathname === "/print/"){
-              return (
-                <ButtonContainer>
-                 <ButtonLinkGreen to="/screen-2/">Return to Results</ButtonLinkGreen>
-                 <ButtonLink to="/screen-2a/">Continue to Benefit Formula</ButtonLink>
-                </ButtonContainer>
-              )
-            }
-            if(index === -1){
-              return null;
-            }
-            if(index === LINKSPATH.length -1){
-              return (
-              <ButtonContainer>
-              <ButtonLinkGreen to={LINKSPATH[index -1].path}>
-               {`Previous: ${LINKSPATH[index -1].label[0] + LINKSPATH[index -1].label.slice(1).toLowerCase()}`}
-              </ButtonLinkGreen>
-              <ButtonLink to="/">Go Home</ButtonLink>
-              </ButtonContainer>
-              )
-            }
-            if(index === 0 ){
-              return (
-              <ButtonContainer>
-              <ButtonLink to="/prescreen-1a/">Get Started</ButtonLink>
-              </ButtonContainer>
-              )
+            const index = LINKSPATH.findIndex(path => (
+              path.path === location.pathname
+            ));
+            let labelLeft, labelRight, urlLeft, urlRight;
+
+            if (index === -1) return;
```

Comment:
Because we can't test this (a route that is not in LINKSPATH) right now, let's put this return back to null to be safe. Sometimes react is upset about blank returns. Probably not too critical though.
```suggestion
            if (index === -1) { 
              return null;
            }
```